nexus-repository-p2-it/src/test/java/org/sonatype/nexus/repository/p2/internal/fixtures/P2RepoRecipes.groovy

```diff
@@ -0,0 +1,39 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2018-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype
+ * .com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License
+ * Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are
+ * trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.nexus.repository.p2.internal.fixtures
+
+import javax.annotation.Nonnull
+import org.sonatype.nexus.repository.Repository
+import org.sonatype.nexus.repository.config.Configuration
+import org.sonatype.nexus.testsuite.testsupport.fixtures.ConfigurationRecipes
+import groovy.transform.CompileStatic
+
+/**
+ * Factory for p2 {@link Repository} {@link Configuration}
+ */
+@CompileStatic
+trait P2RepoRecipes
+    extends ConfigurationRecipes
+{
+  @Nonnull
+  Repository createP2Proxy(final String name,
+                             final String remoteUrl)
```

Comment:
broken formating on line 33

Reply:
in: https://github.com/sonatype-nexus-community/nexus-repository-p2/pull/17/commits/94912ebe5ebec6d38ce4e6c99a076d6a7b89a042